.circleci/config.yml

```diff
@@ -386,24 +337,15 @@ jobs:
             pipenv run pytest --cov=nucypher --cov-report xml:reports/coverage.xml $(circleci tests glob "tests/blockchain/eth/contracts/**/test_*.py" | circleci tests split --split-by=timings)
       - capture_test_results
 
-  config:
+  basics:
```

Comment:
Heh - that's a funny lump of tests.  What made you group these together?

Reply:
I tried to reduce the number of containers. Previous jobs bundled in basics wasted a total of 4 or 5 containers, and now it's reduced to only one. They were short test modules abreast anyway. My heuristic was to get all jobs to last for at least 1 minute; otherwise, they were sufficiently small to be bundled.

Reply:
Gotcha - so more or less these are all the fastest tests.